app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -179,3 +157,60 @@
     display: block;
   }
 }
+
+@media print {
+  // This below styles are for printing out the results on our service/search results page.
+
+  // This sets visibility hidden on all elements except for the searchResultsContainer
+  // and its children to avoid printing irrelevant sections of the page.
+  // This follows the example of this answer https://stackoverflow.com/a/2618980
+  body {
+    visibility: hidden;
+  }
+
+  .searchResultsContainer {
+    visibility: visible;
+  }
+
+  .searchResultsContainer {
```

Comment:
These are the same CSS class, right? These two CSS blocks can be combined.

Reply:
Oops, good catch.